cmd/cli/commands/orders.go

```diff
@@ -13,22 +13,22 @@ var (
 )
 
 func init() {
-	marketSearchCmd.PersistentFlags().Uint64Var(&ordersSearchLimit, "limit", 10, "Orders count to show")
+	ordersListCmd.PersistentFlags().Uint64Var(&ordersSearchLimit, "limit", 10, "Orders count to show")
 
-	marketRootCmd.AddCommand(
-		marketSearchCmd,
-		marketShowCmd,
-		marketCreateCmd,
-		marketCancelCmd,
+	ordersRootCmd.AddCommand(
+		ordersListCmd,
```

Comment:
Everything is `order*`, except this. Why?

Reply:
Because I'm pizdoglazy mudilo